src/atoms/button/Button.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// Lodash
+import _noop from 'lodash/noop';
+
+//Styles
+import "./Button.css"
+
+const Button = ({btnType, btnText, onClick}) => {
+
+    return (
+      <button type={btnType} className="button" onClick={onClick}>{btnText}</button>
+    )
+};
+
+Button.propTypes = {
+  btnType: PropTypes.string,
```

Comment:
Type is still taken as default, but not the same case with text, it should be a required prop wherever this button is used.